src/profile-logic/process-profile.js

```diff
@@ -581,6 +642,9 @@ function _processThread(
 
   const { libs, pausedRanges, meta } = processProfile;
   const { shutdownTime } = meta;
+  const extensions = _processExtensions(
+    mainProcessProfile.meta.extensions || EMPTY_EXTENSIONS
```

Comment:
I would prefer `EMPTY_EXTENSIONS` to be a default parameter on `_processExtensions`